src/app/components/CreateLifeComponent.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Col, Row } from 'antd';
+import { Formik, Field, Form } from 'formik';
+import { useCreateLifeMutation, CreateLifeMutationVariables } from '../api/index';
+
+interface LifeValues {
+    firstName: string;
+    lastName: string;
+    title: string;
+    description: string;
+    birthday: Date;
+    hobbies: string[];
+}
+
+const CreateLifeComponent = () => {
+    const [mutation] = useCreateLifeMutation();
+    const handleCreateLife = async (lifeValues: LifeValues) => {
```

Comment:
you may want to wrap this in a `useCallback`

Reply:
Wrapped it with `useCallback` with empty dependency array. My rationale for an empty array is that there is no need for the return of a new function object between re-renders.